frontend/modules/model/model.sagas.ts

```diff
@@ -72,8 +72,15 @@ export function* downloadModel({ teamspace, modelId }) {
 	}
 }
 
-export function* onModelStatusChanged({ modelData: { status }, teamspace, project, modelId }) {
-	yield put(TeamspacesActions.setModelUploadStatus(teamspace, project, modelId, modelStatusesMap[status]));
+export function* onModelStatusChanged({ modelData, teamspace, project, modelId }) {
+	yield put(TeamspacesActions.setModelUploadStatus(teamspace, project, modelId, uploadFileStatuses[modelData.status]));
+
+	if (status === uploadFileStatuses.ok) {
+		yield put(SnackbarActions.show('Model uploaded succesfully'));
+	}
+	if (status === uploadFileStatuses.failed) {
+		yield put(SnackbarActions.show('Failed to import model'));
```

Comment:
We need to display the error reason somewhere. So if we replace what was the message displayed on the model row to here, we need to display the error message here.